src/python/pcmdi/scripts/pcmdi_metrics_driver.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/igcmg/database/PMP_install/PCMDI_METRICS/bin/python
```

Comment:
This custom path shebang will need removing - I'd note that you can setup your environment variables so a `which python` will return the correct PMP one by `source ~MetricsInstallDir/bin/setup_runtime.sh or .csh`